src/components/ui/TagInput/Tag.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Icon from 'components/ui/Icon';
+import __spacing from 'settings/__spacing';
+import { fontSizes } from 'settings/__fonts';
+
+export interface TagProps {
+  handleClick: React.EventHandler<React.SyntheticEvent>;
+  tagValue: string;
+}
+
+export default function Tag(props: TagProps) {
+  const { tagValue, handleClick } = props;
+  return (
+    <Tag.Wrapper>
+      <span>{tagValue}</span>
+      <i onClick={handleClick}>
```

Comment:
this `<i>` component her is unnecessary as the Icon component already uses an `<i>`. I suggest using a span and adding an onclick on the span or even better extend the Icon to accept onClick events